tic/registers.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2024 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package tic
+
+import (
+	"encoding/binary"
+)
+
+// getVar8 reads an 8 bit value from the Tic at a given register offset.
+func (d *Dev) getVar8(offset Offset) (uint8, error) {
```

Comment:
FYI, you could have used https://periph.io/x/conn/v3/mmr#Dev8 which takes care of all of this.

Reply:
I wasn't aware of that package - very useful! However in this case, two bytes are sent to the device (`cmdGetVariable` + `offset`), so I'm struggling to see I'd use `mmr` to do this.